tools/CACertificates/README.md

```diff
@@ -78,22 +78,30 @@ On Azure IoT Hub, navigate to the "Device Explorer".  Add a new device (e.g. `my
 Note that if you're using this certificate as a DPS registration ID, the ID **must be lower case** or the server will reject it.
 
 ### **PowerShell**
-#### IoT Leaf Device
-* Run `New-CACertsDevice mydevice` to create the new device certificate.
```

Comment:
Both old and new end-to-end tests call `New-CACertsDevice` (and `./certGen.sh create_edge_device_certificate`). It looks like those will continue to work, but should the tests be updated to use the "recommended" approach?

Reply:
Correct. Longer term the E2E tests should be migrated over to the newer commands and the older one can simply be removed.

Reply:
Agree it should be done separately.